### PR TITLE
fix for camera

### PR DESCRIPTION
**Pull request by gieljnssns**
fix for https://github.com/home-assistant/home-assistant/issues/15047#issuecomment-404087158
fix for:
```
2018-07-10 16:10:02 ERROR (MainThread) [homeassistant.components.binary_sensor] Error while setting up platform netatmo
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/helpers/entity_platform.py", line 129, in _async_setup_platform
    SLOW_SETUP_MAX_WAIT, loop=hass.loop)
  File "/usr/lib/python3.5/asyncio/tasks.py", line 400, in wait_for
    return fut.result()
  File "/usr/lib/python3.5/asyncio/futures.py", line 293, in result
    raise self._exception
  File "/usr/lib/python3.5/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/components/binary_sensor/netatmo.py", line 74, in setup_platform
    if not data.get_camera_names():
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/components/netatmo.py", line 84, in get_camera_names
    self.update()
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/util/__init__.py", line 319, in wrapper
    result = method(*args, **kwargs)
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/components/netatmo.py", line 115, in update
    self.camera_data = lnetatmo.CameraData(self.auth, size=100)
  File "/srv/homeassistant/lib/python3.5/site-packages/smart_home/Camera.py", line 79, in __init__
    self.default_camera = list(self.cameras[self.default_home].values())[0]
IndexError: list index out of range
```

### CODE REVIEW

File: smart_home/Camera.py [hunk 1]
```diff
@@ -474,7 +476,7 @@ def moduleMotionDetected(self, module=None, home=None,
                     return True
         elif (self.lastEvent[cam_id]['type'] == 'tag_big_move' or
               self.lastEvent[cam_id]['type'] == 'tag_small_move') and\
-              self.lastEvent[cam_id]['module_id'] == mod_id:
+                self.lastEvent[cam_id]['module_id'] == mod_id:
```
Comment (jabesq) [inline smart_home/Camera.py]: unneeded whitespace

File: smart_home/Camera.py [hunk 2]
```diff
@@ -498,9 +500,9 @@ def moduleOpened(self, module=None, home=None, camera=None, exclude=0):
                 if time_ev < limit:
                     return False
                 elif self.events[cam_id][time_ev]['type'] == 'tag_open' and\
-                     self.events[cam_id][time_ev]['module_id'] == mod_id:
+                        self.events[cam_id][time_ev]['module_id'] == mod_id:
```
Comment (jabesq) [inline smart_home/Camera.py]: unneeded whitespace

File: smart_home/Camera.py [hunk 3]
```diff
@@ -498,9 +500,9 @@ def moduleOpened(self, module=None, home=None, camera=None, exclude=0):
...
                     return True
         elif self.lastEvent[cam_id]['type'] == 'tag_open' and\
-             self.lastEvent[cam_id]['module_id'] == mod_id:
+                self.lastEvent[cam_id]['module_id'] == mod_id:
```
Comment (jabesq) [inline smart_home/Camera.py]: unneeded whitespace

**Review by jabesq — CHANGES_REQUESTED** [review]
Small changes needed, but looks ok to me.
You will also need to rebase on top of tree

File: smart_home/Camera.py (hunk 1 above)
Comment (gieljnssns) [reply smart_home/Camera.py]: When I do that, I get 
```
Error	Flake8	E127 — continuation line over-indented for visual indent	479:15
```

File: smart_home/Camera.py (hunk 2 above)
Comment (gieljnssns) [reply smart_home/Camera.py]: When I do that, I get 
```
Error	Flake8	E127 — continuation line over-indented for visual indent	503:22
```

File: smart_home/Camera.py (hunk 3 above)
Comment (gieljnssns) [reply smart_home/Camera.py]: When I do that, I get 
```
Error	Flake8	E127 — continuation line over-indented for visual indent	506:14
```

**Comment by gieljnssns** [comment]
What will the new version number be?
So I can make a PR to HA...

**Comment by gieljnssns** [comment]
> You will also need to rebase on top of tree

I hope I did it right

**Review by jabesq — APPROVED** [review]

**Comment by jabesq** [comment]
There is still some conflict, I will do the rebase manually.
This will be release in version 1.1.1

**Comment by jabesq** [comment]
Merge done manually

**Comment by gieljnssns** [comment]
> There is still some conflict, I will do the rebase manually.


Thanks

**Comment by Dennydd** [comment]
Have new progress for this?i have HassOs whith Home Assistant 0.74.0 but the camera still does not work

**Pull request closed without merging**